api/HTMLElement.json

```diff
@@ -2242,10 +2254,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3"
+              "version_added": "9"
```

Comment:
Can you split out the `onfoo` changes and update them together with the `foo_event` entries which are sprinkled around?